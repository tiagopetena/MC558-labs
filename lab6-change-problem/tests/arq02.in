11 5882
2 5 1
73 191 1
33 48 2
79 509 3
70 41 3
46 150 3
87 936 2
89 722 1
76 381 1
37 199 2
19 471 3
