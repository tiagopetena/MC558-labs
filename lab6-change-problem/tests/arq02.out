1196 7178
